repository/RediStick-Core.package/RsRediStick.class/instance/shoudAuthenticate.st

```diff
@@ -0,0 +1,3 @@
+actions
+shoudAuthenticate
+	^ password isNotNil 
```

Comment:
`password notNil` is better. `isNotNil` is Pharo specific and loses portability to other smalltalk dialects.